### DISCUSSION TITLE
zh/demo/download

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#686] [Announcements]
# zh/demo/download

分类 V版本 下载地址 (https://github.com/Cotton-Buds/calculator/releases) F版本 下载地址 (https://github.com/Cotton-Buds/colorpicker/releases) 如何选择对应的版本？ 选择对应的版本应该考虑以下几个因素： F系列 在线验证版本 由 @strig...

https://korepi.com/zh/demo/download.html

**Comment by Micah123321** [comment]
测试评论
